### PR TITLE
Arch User Repository: Fix sourceUrl

### PR DESCRIPTION
**Pull request by MrChrisW**
Fixing #2093

### CODE REVIEW

File: share/spice/aur/aur.js [hunk 1]
```diff
@@ -16,7 +21,7 @@
             data: api_result.results,
             meta: {
                 sourceName: "aur.archlinux.org",
-                sourceUrl: 'http://aur.archlinux.org/rpc.php?type=search&arg=' + api_result.name
+                sourceUrl: 'https://aur.archlinux.org/packages/?K=' + query
```
Comment (moollaza) [inline share/spice/aur/aur.js]: Why use the query instead of the package name?

Comment (MrChrisW) [inline share/spice/aur/aur.js]: @moollaza There's no single "package name" returned, if multiple results are found for the query, so `api_result.name` won't work.

This wouldn't even work for a single returned result as the package name is under the results array.

Checkout an example API response:
https://aur.archlinux.org/rpc.php?type=search&arg=firefox

Comment (moollaza) [inline share/spice/aur/aur.js]: :+1:

**Comment by moollaza** [comment]
LGTM :+1:

**Pull request merged**